src/components/SiteDetails.js

```diff
@@ -11,33 +11,37 @@ const checklistItems = [
  * Component to show details about Frogtoberfest in the homepage.
  */
 const SiteDetails = () => (
-  <div className="rounded mx-auto shadow w-3/4 sm:w-1/2 lg:w-1/3">
-    <div className="px-6 py-4">
-      <div className="font-bold mb-4">
-        <p className="text-lg mb-1">
-          Frogtoberfest, inspired from Hacktoberfest is a month-long open source contribution challenge for
-          Leapfroggers.
-        </p>
+  <div className="md:py-4 SiteDetails">
+    <div className="rounded mx-auto shadow w-3/4 sm:w-1/2">
+      <div className="px-6 py-4">
+        <div className="font-bold mb-4">
+          <p className="SiteDetails__text text-lg mb-1">
```

Comment:
Instead of using our own class name, we should instead leverage `tailwind.css` classes here.  It is an utility first library which gives us all the utilities for margin, padding, width and everything else? Can we do that instead?

Read more about it here: https://tailwindcss.com/docs/utility-first/

Reply:
Sorry that we were not clear about the approach in the first place.

Reply:
Sure @mesaugat